### DISCUSSION TITLE
How to change password in mnesia plugin?

### DISCUSSION THREAD

**Discussion by DatafiSystems** [emqx/emqx#8954] [Q&A] [answered]
I am having a problem getting changes to reflect.

The process I have followed from Docker install 4.3.18

Edit emqx.comf to:
`allow_anonymous = false`

Edit emqx_auth_mnesia.conf to: (uncomment what is there)
`auth.user.1.username = admin`
`auth.user.1.password = public`

then run `emqx_ctl plugins load emqx_auth_mnesia`

##

Now if I want to change that password, not matter what I do hasn't seemed to work.

Change emqx_auth_mnesia.conf to:
`auth.user.1.username = admin`
`auth.user.1.password = publictest`

then run `emqx_ctl plugins reload emqx_auth_mnesia`

When logging into the broker I can still use the old password? I have tried unloading and loading, reloading. Nothing seems to work.

TIA. Any guidance is welcome.

**Comment by lafirest** [comment, marked as answer]
@DatafiSystems 
when you enabled the `emqx_auth_mnesia`, you can use `emqx_ctl acl` command to manage it
<img width="1391" alt="image" src="https://user-images.githubusercontent.com/22971499/190989794-0cd5f5c0-e08c-4f1d-8eb6-e1dc689cc34d.png">
